pkg/trimaran/targetloadpacking/targetloadpacking_test.go

```diff
@@ -412,3 +436,13 @@ func getNodes(nodesNum int64) (nodes []*v1.Node) {
 	}
 	return
 }
+
+func NewFramework(fns []st.RegisterPluginFunc, args []config.PluginConfig, opts ...runtime.Option) (framework.Framework, error) {
```

Comment:
@Huang-Wei Added this function. Ideally this should replace `NewFramework` in `pkg/scheduler/testing/framework_helpers.go` to allow plugin args.